android/src/main/java/com/clevertap/react/CleverTapEvent.kt

```diff
@@ -19,7 +19,10 @@ enum class CleverTapEvent(val eventName: String, val bufferable: Boolean = false
     CLEVERTAP_PUSH_NOTIFICATION_CLICKED("CleverTapPushNotificationClicked", bufferable = true),
     CLEVERTAP_ON_PUSH_PERMISSION_RESPONSE("CleverTapPushPermissionResponseReceived"),
     CLEVERTAP_ON_VARIABLES_CHANGED("CleverTapOnVariablesChanged"),
-    CLEVERTAP_ON_VALUE_CHANGED("CleverTapOnValueChanged");
+    CLEVERTAP_ON_VALUE_CHANGED("CleverTapOnValueChanged"),
+    CLEVERTAP_CUSTOM_TEMPLATE_PRESENT("CleverTapCustomTemplatePresent", bufferable = true),
+    CLEVERTAP_CUSTOM_FUNCTION_PRESENT("CleverTapCustomFunctionPresent", bufferable = true),
+    CLEVERTAP_CUSTOM_TEMPLATE_CLOSE("CleverTapCustomTemplateClose");
```

Comment:
`CLEVERTAP_CUSTOM_TEMPLATE_CLOSE `should this be `CLEVERTAP_CUSTOM_TEMPLATE_CLOSED`

`CLEVERTAP_CUSTOM_TEMPLATE_PRESENT` should this be `CLEVERTAP_CUSTOM_TEMPLATE_PRESENTED`

Reply:
Let's keep the current names, I think they are more clear. The present tense signifies that the JS handler should act on those events and their action have not been completed. TEMPLATE_PRESENTED looks like the template is already presented, but it is actually the JS handler job to present it.

Reply:
Makes sense